core/validation/messages.py

```diff
@@ -37,5 +37,6 @@
 MISSING_OTHER_FUNDING_SOURCES = (
     "You’ve not entered any Other Funding Sources. You must enter at least 1 over all projects."
 )
+NEGATIVE_NUMBER = "You've entered a negative number. Please enter a positive number."
```

Comment:
Just checking you're running / have ran this by design?

Reply:
Yeh, I've sent a message, will amend accordingly if needed

Reply:
Updated & signed-off by design